fortify-ssc-parser-util-cyclonedx/src/main/java/com/fortify/ssc/parser/cyclonedx/domain/Vulnerability.java

```diff
@@ -70,13 +70,25 @@ public final Priority getFortifyPriority() {
 		return getMaxSeverity().getFortifyPriority();
 	}
 	
+	public final VulnerabilityRatingMethod getVulnMethod() {
```

Comment:
You're first iterating over the array here, and then again in getVulnScore() and getVulnVector() methods. Even though the ratings array usually contains only a small number of entries, it's more efficient to have a Java method that returns the appropriate (full) VulnerabilityRating object, and then just get rating method, score and vector from that VulnerabilityRating object instead of iterating over the array multiple times.